README.md

```diff
@@ -246,6 +246,9 @@ We basically just have to define a `work` function/object/lambda whose signature
 
 ```c++
 int main(void) {
+  //when we hear SIGTERM, wait 28 seconds to drain traffic then wait a second for the threads to stop
```

Comment:
great question! its because a lot of systems use 30 seconds as the default, so to avoid SIGKILL we want to make sure we are done with everything before 30 seconds, so i wanted to have 29 seconds in total, drain(28) + shutdown(1) == 29 < 30

Reply:
I might've missed it, but does `prime_server` stop accepting new connections once it goes into the `draining` state?